portal/static/js/main.js

```diff
@@ -1825,7 +1827,42 @@ OrgTool.prototype.getHereBelowOrgs = function() {
 var OT = new OrgTool();
 
 var tnthAjax = {
+    "getStillNeededCoreData": function(userId, sync, callback) {
+        if (!hasValue(userId)) return false;
+        $.ajax ({
+            type: "GET",
+            url: "/api/coredata/user/" + userId + "/still_needed",
+            cache: false,
+            async: (sync ? false : true)
+        }).done(function(data) {
+            if (data && data.still_needed) {
+                if (callback) callback(data.still_needed);
+            } else {
+                if (callback) callback({"error": "no data returned"});
```

Comment:
isn't this misleading in the case of a well functioning call, and there's zero items in the still_needed list?